node/src/main.rs

```diff
@@ -719,12 +721,22 @@ fn start_block_ingestor(
                 "network_name" => &network_name
             );
             let eth_adapter = eth_adapters.cheapest().unwrap(); //Safe to unwrap since it cannot be empty
-            let block_ingestor = BlockIngestor::new(
-                chain_store,
-                eth_adapter.clone(),
+            let chain_logger = logger_factory.component_logger(
+                "BlockIngestor",
+                Some(ComponentLoggerConfig {
+                    elastic: Some(ElasticComponentLoggerConfig {
+                        index: String::from("block-ingestor-logs"),
+                    }),
+                }),
+            )
+            .new(o!("provider" => eth_adapter.provider().to_string()));
+
+            let chain = ethereum::Chain::new(chain_logger.clone(), chain_store, eth_adapter.clone(), *ANCESTOR_COUNT);
```

Comment:
I am going light on how exactly setup/early startup works, since I know that that will need to be revisited in future PR's. Totally agree that it's a bit weird